.github/workflows/build-images.yaml

```diff
@@ -48,7 +62,9 @@ jobs:
           tags: ${{ env.IMG_TAGS }}
           platforms: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
           build-args: |
-            version=${{ env.VERSION }}
+            GIT_SHA=${{ env.GIT_SHA }}
+            version=${{ env.version }}
+            DIRTY=${{ env.DIRTY }}
```

Comment:
nitpick: Can we please keep the case consistent here?

Reply:
We could, but there is reasoning behind why `version` is lowercase in comparison to the other build args... `VERSION` is already defined in the [parent Dockerfile](https://catalog.redhat.com/software/containers/ubi9/go-toolset/61e5c00b4ec9945c18787690?container-tabs=dockerfile).  

I made the choice of making that one lowercase as was seen in the previous version of the Dockerfile. I suppose since one of the args is already lowercase, we can make all lowercase so it's less confusing.